src/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -289,7 +284,7 @@ def create(dataset, target, feature, max_iterations=10,
 
     state = {
         '_class_label_to_id': class_label_to_id,
-        '_custom_classifier': custom_NN,
+        '_custom_classifier': mxnet_model.custom_NN,
```

Comment:
This exposes MXNet to the toolkit. Let's just store the abstract wrapper class here, and then change all the places that use `_custom_classifier` to work with that

Reply:
Do you mean we pass object from the class here i.e. `mxnet_model` or the class itself i.e. `MultiLayerPerceptronMXNetModel()`?